### DISCUSSION TITLE
Network drive location

### DISCUSSION THREAD

**Discussion by ayush-lal** [alefragnani/vscode-project-manager#454] [Q&A] [answered]
Hi guys, Does VSCode Project Manager support network drive locations? 

I have tried to add the following line to the Projects.json file however it doesnt seem to work:
`"rootPath":  "\\Domain\Folder"`

**Comment by alefragnani** [comment, marked as answer]
Hi @ayush-lal ,

Yes, you can use network drive locations. Since you are doing it _by hand_, you just have to _escape_ the backslash sign .

```json
  "rootPath": "\\\\Domain\\Folder"
```

If you open the location in VS Code and use the `Project Manager: Save Project` command, it will handle it to you

Hope this helps

**Reply by ayush-lal** [reply]
Ah awesome! Thanks so much for your help :)
